### ISSUE TITLE
Self compiled Proton does not compile 32 bit Wine

### ISSUE THREAD

**Issue by minecraft2048** [ValveSoftware/Proton#2263]
I use the Vagrant step as explained in the README, then Proton builds with no errors, and when I copied the files into Steam as explained in the Install Proton Locally step, every games that runs with that Proton installation crashes with 
```
Traceback (most recent call last):
  File "/home/feanor/.local/share/Steam/compatibilitytools.d/proton-localbuild/proton", line 306, in <module>
    shutil.copy(basedir + "/dist/lib/wine/fakedlls/vrclient.dll", dst)
  File "/usr/lib/python3.7/shutil.py", line 245, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.7/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/home/feanor/.local/share/Steam/compatibilitytools.d/proton-localbuild/dist/lib/wine/fakedlls/vrclient.dll'
```

There is nothing at dist/lib/wine except dxvk folder, while Proton 3.16 beta provided from Steam have a lot of dlls in it

**Comment by f0rmatme** [comment]
i had this issue too, i think only wine64 gets compiled and not wine for 32 bit.

**Comment by minecraft2048** [comment]
I can't check where this bug is introduced as if I build later than commit 55b80e0c68c6a545611b02b286c2114a9b371d8b compile fails with missing gnutls26 headers

**Comment by minecraft2048** [comment]
At the build folder and after looking at the makefiles I noticed that `obj-wine32` and `dist-wine32` is missing, but the compile does finish without errors

**Comment by minecraft2048** [comment]
Workaround:
Running `make dist` directly doesn't work, as it will not compile 32 bit Wine. What do work for me is to run `make all`, then wait until it says run gmake and it crashes with error 2, then run gmake again and again until it doesn't crash with error 2, then run `make install` as usual.

**Comment by aeikum** [comment]
Thanks for reporting, will have a fix for the `dist` target in the next release. In the meantime, using `make all` as your first target seems like a decent workaround.

**Comment by aeikum** [comment]
Hello, I believe this was fixed with the 3.16-7 release. Please retest and let me know if problems linger. Note that you'll need to reconfigure if you are using the steam runtime build option (default) as we now require a path to an end-user runtime.
